web/pdf_page_view.js

```diff
@@ -173,20 +464,15 @@ class PDFPageView {
     this.#annotationMode =
       options.annotationMode ?? AnnotationMode.ENABLE_FORMS;
     this.imageResourcesPath = options.imageResourcesPath || "";
+    this.enableDetailCanvas = AppOptions.get("enableDetailCanvas");
```

Comment:
Sorry, but this isn't the "correct" way to handle options (note that the `maxCanvasPixels` case below is special for compatibility reasons).

Hence, please instead add the option in https://github.com/mozilla/pdf.js/blob/fbac4ab5f35dd4556069b69af51f78dbaaa62229/web/app.js#L471-L503 the same way that all existing `PDFViewer`/`PDFPageView` options are handled.